pkgs/applications/networking/instant-messengers/zulip-term/pytest-executable-name.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/tests/cli/test_run.py b/tests/cli/test_run.py
+index 459aa82..c6e434e 100644
+--- a/tests/cli/test_run.py
++++ b/tests/cli/test_run.py
+@@ -180,7 +180,7 @@ def test_main_multiple_autohide_options(capsys, options):
+         assert str(e.value) == "2"
+     captured = capsys.readouterr()
+     lines = captured.err.strip('\n')
+-    lines = lines.split("pytest: ", 1)[1]
```

Comment:
any plan to upstream ?

Reply:
I considered it, but I think the name is specific to our pytest infrastructure. I could split it using some regex.